### ISSUE TITLE
Interfacing with NDR methods

### ISSUE THREAD

**Issue by sa-lee** [sa-lee/liminal#2]
Multiple low dimensional views as a nested data.frame. Search over a parameter space.

Target interface:

```{r}
limn_embed(.data, cols, .ndr_fun, .params, .parallel )
```
where cols are the data that will be embedded, .ndr_fun is the ndr function, .params is the grid of parameters, and .parallel specifies how to compute the embeddings in parallel. This allows limn to plug in an ndr method and store the results as a nested data.frame

**Comment by sa-lee** [comment]
Out of scope
